src/session/content/message_row/mod.rs

```diff
@@ -357,7 +359,11 @@ impl MessageRow {
         let content_ref = self.imp().content.borrow();
         let content = content_ref.as_ref().unwrap();
 
-        if is_outgoing {
+        // MessageSticker should handle their size and position internally.
+        //In the future the same exception should be made for MessageVideoNote
```

Comment:
True, I also saw another use for message events (for when they will actually be message widgets). I think I want to work on this before merging this PR.